zboxcore/sdk/dirworker.go

```diff
@@ -50,14 +61,19 @@ func (req *DirRequest) createDirInBlobber(blobber *blockchain.StorageNode) error
 	body := new(bytes.Buffer)
 	formWriter := multipart.NewWriter(body)
 	formWriter.WriteField("connection_id", req.connectionID)
-	formWriter.WriteField("dir_path", filepath.Join("/", req.name))
+
+	dirPath := filepath.ToSlash(req.name)
+	if !strings.HasPrefix(dirPath, "/") {
+		dirPath = filepath.Join("/", dirPath)
+	}
```

Comment:
This check is not needed, even if `dirPath` has prefix "/", `filepath.Join("/", dirpath)` would have only `/`. Ex: https://go.dev/play/p/ZCevmdrs_qg.

Reply:
fixed it